data/sql/updates/pending_db_world/rev_1765988701219696700.sql

```diff
@@ -0,0 +1,34 @@
+-- Lykul Wasp
+DELETE FROM `creature_template_model` WHERE (`CreatureID` = 17732);
+INSERT INTO `creature_template_model` (`CreatureID`, `Idx`, `CreatureDisplayID`, `DisplayScale`, `Probability`, `VerifiedBuild`) VALUES
+(17732, 0, 18722, 1, 1, 12340),
```

Comment:
Small thing, but if the changes aren't verified by a sniff, try not to put 12340 as the VerifiedBuild. We're really trying to reduce that as much as we can these days. 😛 But I will check brute and try and corroborate these changes shortly.